osmaxx/excerptexport/management/commands/purge_old_result_files.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import time
+
+from django.utils import timezone
+
+from django.core.management.base import BaseCommand
+
+from osmaxx.excerptexport._settings import OLD_RESULT_FILES_REMOVAL_CHECK_INTERVAL_HOURS, PURGE_OLD_RESULT_FILES_AFTER
+from osmaxx.excerptexport.models import OutputFile
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = 'removes old output-files from exports after the specified interval' \
+           ' "PURGE_OLD_RESULT_FILES_AFTER" setting' \
+           ' - runs until interrupted every "OLD_RESULT_FILES_REMOVAL_CHECK_INTERVAL_HOURS" unless' \
+           '--run_once option is given'
+
+    can_import_settings = True
+
+    def add_arguments(self, parser):
+        parser.add_argument('--run_once', action='store_true')
+
+    def handle(self, *args, **options):
+        if options.get('run_once', False):
+            self._run()
+        else:
+            while True:
+                self._run()
+                time.sleep(OLD_RESULT_FILES_REMOVAL_CHECK_INTERVAL_HOURS.total_seconds())
+
+    def _run(self):
+        too_old = timezone.now() - PURGE_OLD_RESULT_FILES_AFTER
+        self._success(
+            "Removing old output files that are older than {}".format(
+                too_old.strftime("%Y-%m-%d %H:%M:%S")
+            )
+        )
+        self._remove_old_files()
+
+    def _success(self, message):
+        self.stdout.write(
+            self.style.SUCCESS(message)
+        )
+
+    def _remove_old_files(self):
+        try:
+            too_old = timezone.now() - PURGE_OLD_RESULT_FILES_AFTER
```

Comment:
This constant name reads very nice when setting its value

> Purge old result files after 14 days. &rarr; `PURGE_OLD_RESULT_FILES_AFTER = timedelta(days=14)`

but here, when used in the code, it doesn't play out so well.

Not sure what it should be called instead. `RESULT_FILE_RETAINMENT_DURATION` is the best I can come up with right now.